independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus-extension/code/quarkiverse/java/.github/workflows/release.yml

```diff
@@ -77,10 +77,3 @@ jobs:
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
           branch: ${{github.base_ref}}
-
-      - name: Push tags
-        uses: ad-m/github-push-action@v0.6.0
```

Comment:
Couldn't we get rid of it entirely? My understanding was that with the new version of `actions/checkout`, we would be able to push with `git` directly?

Reply:
Hm, maybe yes, I'll need to test before doing that

Reply:
Done